manifests/o/o2sh/onefetch/2.12.0/o2sh.onefetch.locale.en-US.yaml

```
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS0.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: o2sh.onefetch
+PackageVersion: 2.12.0
+PackageLocale: en-US
+Publisher: Ossama Hjaji
+PublisherUrl: https://github.com/o2sh/
+PublisherSupportUrl: https://github.com/o2sh/onefetch/issues
+# PrivacyUrl: 
+Author: Ossama Hjaji
+PackageName: onefetch
```

```suggestion
PackageName: Onefetch
```
This must match Control Panel (i.e. capitalization of the `o`)

---

I have updated the installer to match the manifest.